config/webpack/webpack.config.dev.js

```diff
@@ -2,11 +2,14 @@
 
 var webpack = require("webpack");
 var config = require("./webpack.config");
+var LodashModuleReplacementPlugin = require("lodash-webpack-plugin");
 
 // **WARNING**: Mutates base configuration.
 // We do this because lodash isn't available in `production` mode.
 config.output.filename = config.output.filename.replace(/\.min\.js$/, ".js");
 config.plugins = [
+  new LodashModuleReplacementPlugin(),
+  new webpack.optimize.OccurenceOrderPlugin(),
```

Comment:
Let's _not_ do this. It often is worse / no op for `min + gz` and muddles things up.